packages/client/src/pages/MainPage/MainPage.tsx

```diff
@@ -1,5 +1,45 @@
-import { FunctionComponent } from 'react';
-
-const MainPage = () => <h1>Main Page</h1>;
-
-export default MainPage as FunctionComponent;
+import React, { FunctionComponent, useState } from 'react';
+import styles from './mainPage.module.scss';
+import { Button, Switch } from 'components';
+import Logo from 'assets/img/Mainpage.svg';
+import { ROUTES } from 'router';
+import { Link } from 'react-router-dom';
+
+const MainPage = () => {
+  const [theme, toggleTheme] = useState(false);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.header}>
+        <span>{theme ? 'Светлая тема' : 'Темная тема'}</span>
+        <Switch
+          checked={theme}
+          onChange={() => {
+            toggleTheme(!theme);
+          }}></Switch>
```

Comment:
```suggestion
        <Switch
          checked={theme}
          onChange={() => {
            toggleTheme(!theme);
          }} />
```
И лучше будет вынести onChange в отдельную функцию, инлайн функции не очень хороши для производительности ререндера. Конечно это экономия на спичках, но хорошая практика.